.github/workflows/build_and_test.yml

```diff
@@ -44,8 +56,14 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: coveralls --service=github
-    - name: Build Package
-      run: ./scripts/build.sh ${{ matrix.build_arg }}
+    - name: Make Deploy Directory
+      run: mkdir deploy
+    - name: Build Installer
+      run: pyinstaller --onefile ${{ matrix.pyinstall_args }} --add-data "${{ matrix.add_data }}" epregressions/runner.py
+    - name: Fixup Installer
+      run: |
+        VERSION_STRING=$(grep VERSION epregressions/__init__.py | cut -d= -f2 | xargs)
+        ${{ matrix.pack_tool }} deploy/EnergyPlusRegressionTool-"${VERSION_STRING}"-${{ matrix.os }}${{ matrix.extension }} ${{ matrix.pack_suffix }}
```

Comment:
This section replaces the build script.  Same commands executed, just keeping it right in the CI workflow file.